src/models/luckyTickets/luckyTickets.go

```diff
@@ -0,0 +1,93 @@
+package luckyTickets
+
+import (
+	"fmt"
+	"errors"
+)
+
+func validate(obj *context) (err error){
+	if (obj.min - 99999) < 0 || (obj.max - 99999) < 0 || obj.min > obj.max || obj.min > 999999 || obj.max > 999999{
+		return errors.New("incorrect data!")
+	}
+	return
+}
+type errorInstruction struct {
+	Status string
+	Reason string
+}
+
+type context struct {
```

Comment:
`context` is not a good name here, because Go has a standard package with the same name - https://golang.org/pkg/context/ . See the great video on that https://www.youtube.com/watch?v=LSzR0VEraWw

Reply:
yea, but that name given in the task list.
so, what i need to do?

Reply:
use a name `taskContex` or something like that